packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -40,6 +62,33 @@ export interface AcceleratorProps {
    * @default true
    */
   readonly enabled?: boolean;
+
+  /**
+   * IP addresses associated with the accelerator.
+   *
+   * Optionally, if you've added your own IP address pool to Global Accelerator (BYOIP), you can choose IP
+   * addresses from your own pool to use for the accelerator's static IP addresses when you create an accelerator.
+   * You can specify one or two addresses, separated by a comma. Do not include the /32 suffix.
```

Comment:
I think the statement `, separated by a comma` does not necessary because it is for CFn docs.

```suggestion
   * You can specify one or two addresses. Do not include the /32 suffix.
```

Reply:
I'd like to keep this as-is, as that's the verbiage in the official [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-globalaccelerator-accelerator.html#aws-resource-globalaccelerator-accelerator-return-values).